pep/app/pods/components/document/text/component.ts

```diff
@@ -242,7 +331,23 @@ export default class DocumentText extends Component<DocumentTextArgs> {
      */
     scrollToPage(page: number) {
         const element = this.containerElement?.querySelector(`[data-page-start='${page}']`);
-        element?.scrollIntoView();
+        this.animateScrollToElement(element);
+    }
+
+    /**
+     * Animated the scroll to an element using a specified vertical offset
+     *
+     * @param {(Element | null)} [element]
+     * @memberof DocumentText
+     */
+    animateScrollToElement(element?: Element | null) {
+        const container = this.containerElement?.closest('.page-content-inner');
```

Comment:
i would store this element once on the did-insert of the component instead of finding it each time (probably in the `setupListeners()` action)